### DISCUSSION TITLE
Group Assignment #4 📋

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/24SU-ASIAAM-191A#53] [Group Assignment]
#### Due 7/29

For [Group Assignment #4](https://albertkun.github.io/24SU-ASIAAM-191A/assignments/week5/group_assignment/), create a rough draft of your website, include a `readme.md`, and be prepared to give a short 2-3 minute presentation with your group members. 

### Submission

Post the link to your GitHub repository and website below.

ex.
**Group Repo:** https://github.com/akochaphum/my-awesome-group
**Group website:** https://akochaphum.github.io/my-awesome-group/index.html

**Comment by sjeyakum** [comment]
**Newbury Park Student Athletes' H2Opinions**

Group repo: https://github.com/zhamic7/aa191a-su24
Group website: https://zhamic7.github.io/aa191a-su24/

**Comment by stinest** [comment]
**Community on Campus: Palestinian Solidarity at UCLA**
repo: https://github.com/stinest/24su-191a-group4
website: https://stinest.github.io/24su-191a-group4/mapplication/index.html

**Comment by chrissanrow** [comment]
**SEA LGBTQ+ @ UCLA**

**Group repo:** https://github.com/chrissanrow/191A_LGBTQintl
**Group website:** https://chrissanrow.github.io/191A_LGBTQintl/draft/

**Comment by kianaptran** [comment]
Group ATE: Observations Regarding the Food Prep Storage of UCLA Commuter Students

Group repo: https://github.com/pwinnie2026/group-ate
Group website: https://pwinnie2026.github.io/group-ate/index.html

**Comment by nesslee123** [comment]
Is UCLA Really Accessible?

Group repo: https://github.com/nesslee123/disability-project
Group website: https://nesslee123.github.io/disability-project/Project1/index.html

**Comment by Keiya-Wada** [comment]
JAMsj Japanese American Narrative:

Group repo: https://github.com/kimada62/JAMsjJAnarrative
Group website: https://keiya-wada.github.io/asiaam-labs/JAMsj/

**Comment by jasminereyz** [comment]
The Sankofa-Cation Project 

Group Repo: https://github.com/jasminereyz/Post-Incarceration-Health/tree/main

Group Website : will be posted once committed

**Reply by caitlynn-hauw** [reply]
<img width="970" alt="PNG image" src="https://github.com/user-attachments/assets/35679436-fdd1-4def-a944-639a9fa3124f">
